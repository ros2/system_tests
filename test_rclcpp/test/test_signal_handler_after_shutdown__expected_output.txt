Registered custom signal handler.
Called rclcpp::init.
Waiting to give an opportunity for interrupt...
Calling rclcpp::shutdown...
Called rclcpp::shutdown.
Waiting to give an opportunity for interrupt...
Custom sigint handler called.
Exiting.
