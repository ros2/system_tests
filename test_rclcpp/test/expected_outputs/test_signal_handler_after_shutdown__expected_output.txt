[INFO] [test_signal_handler]: Registered custom signal handler.
[INFO] [test_signal_handler]: Called rclcpp::init.
[INFO] [test_signal_handler]: Waiting to give an opportunity for interrupt...
[INFO] [test_signal_handler]: Calling rclcpp::shutdown...
[INFO] [test_signal_handler]: Called rclcpp::shutdown.
[INFO] [test_signal_handler]: Waiting to give an opportunity for interrupt...
[INFO] [test_signal_handler]: Custom sigint handler called.
[INFO] [test_signal_handler]: Exiting.
